### PR TITLE
Add two utility functions to HighsDomain.cpp

### PR DESCRIPTION
**Pull request by fwesselm**
I added two small utility functions to `src/mip/HighsDomain.cpp` to remove identical code blocks:
1. `computeDelta` for computing change in bounds on constraint activities
2. `boundRange` for computing the bound range.

I also fixed some typos along the way.

### CODE REVIEW

**Review by jajhall — APPROVED** [review]
I have to trust that your rationalisation of the duplicated code is correct. Did you do a MIP benchmark run?

**Comment by fwesselm** [comment]
I only tested on a handful of instances so far. I hope to have the full results by the end of next week, if that's OK.

File: src/mip/HighsDomain.cpp [hunk 1]
```diff
@@ -1352,23 +1328,19 @@ void HighsDomain::computeMaxActivity(HighsInt start, HighsInt end,
...
 
   if (mipsolver->variableType(col) != HighsVarType::kContinuous) {
-    bound = std::floor(double(boundVal + mipsolver->mipdata_->feastol));
-    if (bound < col_upper_[col] &&
-        col_upper_[col] - bound >
-            1000.0 * mipsolver->mipdata_->feastol * std::fabs(bound))
-      accept = true;
-    else
-      accept = false;
+    bound = static_cast<double>(floor(boundVal + mipsolver->mipdata_->feastol));
```
Comment (fwesselm) [inline src/mip/HighsDomain.cpp:1340]: Floor is now computed (using HighsCDouble) before casting back to double. This causes a behavior change on 7 out of 854 problems that I tested.

File: src/mip/HighsDomain.cpp [hunk 2]
```diff
@@ -1397,13 +1369,10 @@ double HighsDomain::adjustedLb(HighsInt col, HighsCDouble boundVal,
...
 
   if (mipsolver->variableType(col) != HighsVarType::kContinuous) {
-    bound = std::ceil(double(boundVal - mipsolver->mipdata_->feastol));
-    if (bound > col_lower_[col] &&
-        bound - col_lower_[col] >
-            1000.0 * mipsolver->mipdata_->feastol * std::fabs(bound))
-      accept = true;
-    else
-      accept = false;
+    bound = static_cast<double>(ceil(boundVal - mipsolver->mipdata_->feastol));
```
Comment (fwesselm) [inline src/mip/HighsDomain.cpp:1372]: Ceiling is now computed (using HighsCDouble) before casting back to double. This causes a behavior change on 7 out of 854 problems that I tested.

**Pull request merged**